docs/user-guide/text-editor-toolbar.md

```diff
@@ -21,9 +21,11 @@ With the *Text Editor Toolbar* it is possible to customize the text by modifying
 
 * **Indent/Outdent** <img src="../img/button/indent.jpg" class="ms-docbutton" style="max-height:15px;"/> to  indent the text in relation to the left margin or to the right margin
 
-* **Link** <img src="../img/button/link.jpg" class="ms-docbutton" style="max-height:15px;"/> to convert in hyperlink the selected portion of text
+* **Link** <img src="../img/button/link.jpg" class="ms-docbutton" style="max-height:15px;"/> to convert in hyperlink the selected portion of text. The editor can define hyperlinks to external web pages by choosing the *External link* option on the **Link target** dropdown menu and then entering a web page URL or define the hyperlinks to other sections of the same story by choosing the section from the same **Link target** dropdown menu. 
```

Comment:
* **Link** <img src="../img/button/link.jpg" class="ms-docbutton" style="max-height:15px;"/> to configure an hyperlink for the selected portion of text. The GeoStory editor can define hyperlinks to external web pages by choosing the *External link* option in the **Link target** dropdown menu and entering the related URL. As an alternative, it is also possible to define an hyperlinks to other sections of the same GeoStory by choosing one of the sections available in the **Link target** dropdown menu.